tests/functional/tests.py

```diff
@@ -2,7 +2,7 @@
 import os
 
 from subprocess import call
-from raven.utils.compat import TestCase
+from raven.utils.testutils import BaseTestCase as TestCase
```

Comment:
Now this refactoring resulted in renaming this class twice (TestCase -> BaseTestCase -> TestCase) which I'm not a fan of but I wanted to keep the changes minimal.